package.json

```diff
@@ -61,16 +64,17 @@
     ]
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
-    "eslint": "^7.32.0",
+    "@typescript-eslint/eslint-plugin": "^5.2.0",
+    "@typescript-eslint/parser": "^5.2.0",
```

Comment:
tried running `yarn lint` to validate these upgrades and it fails for me, does it pass for you?

Reply:
`'place.test.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.`

Reply:
Should have put this in the Gotchas: 
On this branch `place.test.ts` is commented out (done on an earlier branch & pinned to be fixed by someone with a knowledge base to do so). Still waiting on [this PR](https://github.com/normative/dtpr-stickers/pull/48) to be merged before we can test this package update.

Reply:
Ok just commented on that PR so we can get it merged